jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -28,6 +28,15 @@
       },
       "version": "release-0.42"
     },
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/prometheus-operator/prometheus-operator",
+          "subdir": "jsonnet/mixin"
+        }
+      },
+      "version": "master"
```

Comment:
I am pinning to `master` because this doesn't exist on `release-0.42` and prometheus-operator alerts aren't very fast-changing.